lib/components/narrative/default/default-itinerary.js

```diff
@@ -1,34 +1,76 @@
+import moment from 'moment-timezone'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
+import { connect } from 'react-redux'
 
 import NarrativeItinerary from '../narrative-itinerary'
 import ItineraryBody from '../line-itin/connected-itinerary-body'
-import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
-import { getTotalFareAsString } from '../../../util/state'
 
-const { isBicycle, isTransit } = coreUtils.itinerary
-const { formatDuration, formatTime } = coreUtils.time
+import ItinerarySummary from './itinerary-summary'
+
+const { calculateFares, isBicycle, isTransit, isMicromobility } = coreUtils.itinerary
 
+/**
+ * Obtains the description of an itinerary in the given locale.
+ */
 // FIXME move to core utils
 function getItineraryDescription (itinerary) {
   let primaryTransitDuration = 0
-  let mainMode = 'Walk'
+  let accessModeId = 'walk'
   let transitMode
   itinerary.legs.forEach((leg, i) => {
-    const {duration, mode, rentedBike} = leg
+    const {duration, mode, rentedBike, rentedVehicle} = leg
     if (isTransit(mode) && duration > primaryTransitDuration) {
       // TODO: convert OTP's TRAM mode to the correct wording for Portland
       primaryTransitDuration = duration
-      transitMode = mode.toLowerCase()
+      transitMode = <FormattedMessage id={`common.otpTransitModes.${mode.toLowerCase()}`} />
     }
-    if (isBicycle(mode)) mainMode = 'Bike'
-    if (rentedBike) mainMode = 'Bikeshare'
-    if (mode === 'CAR') mainMode = 'Drive'
+    if (isBicycle(mode)) accessModeId = 'bike'
+    if (isMicromobility(mode)) accessModeId = 'micromobility'
+    if (rentedVehicle) accessModeId = 'micromobilityRent'
+    if (rentedBike) accessModeId = 'bikeshare'
+    if (mode === 'CAR') accessModeId = 'drive'
   })
-  let description = mainMode
-  if (transitMode) description += ` to ${transitMode}`
-  return description
+
+  const mainMode = <FormattedMessage id={`common.accessModes.${accessModeId}`} />
+  return transitMode
+    ? <FormattedMessage id='components.DefaultItinerary.multiModeSummary' values={{ accessMode: mainMode, transitMode }} />
+    : mainMode
+}
+
+/**
+ * Formats the given duration according to the selected locale.
+ */
+function formatDuration (duration) {
```

Comment:
I recommend changing this into a proper functional component:
```suggestion
function FormattedDuration ({ duration }) {
```

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2